### ISSUE TITLE
Activity: Header should be one accessible element

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12852]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to Activity and tap on any activity to see it's details.
1. Swipe right to navigate through the UI elements at the top of the activity details screen.

**Result:** the elements at the top of the activity detail such as name, date, role, time should be grouped and ordered in a better way. ([12s](https://cloudup.com/cLSi4c8Db4s))

<img src="https://user-images.githubusercontent.com/1119271/68170070-94678680-ff2b-11e9-8598-5c56275ab7d6.png" width="270" alt="voiceover-activity-detail-header-is-not-grouped-well-wpios-13 5 0 3-app-store-version">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
